src/frontend/src/lib/constants/app.constants.ts

```diff
@@ -57,6 +58,13 @@ export const SIGNER_CANISTER_ID = LOCAL
 		? import.meta.env.VITE_STAGING_SIGNER_CANISTER_ID
 		: import.meta.env.VITE_IC_SIGNER_CANISTER_ID;
 
+export const SIGNER_PAYMENT_TYPE = {
```

Comment:
Will there be more constant? If yes, I would create a new module at the same level and name it for example `signer.constants.ts`

Actually even if there aren't more constant. In `app.constant` it's mostly IDs, env variable mapping or things like seconds. It feels those are quite distinctive in regards to this particular new constant.

Reply:
Done.